Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -742,6 +773,15 @@ static String mutationObserverNotificationString()
     return { };
 }
 
+bool UnifiedPDFPlugin::layerNeedsPlatformContext(const WebCore::GraphicsLayer* layer) const
+{
+#if ENABLE(UNIFIED_PDF_PAINT_SELECTIONS)
```

Comment:
After talking to Simon, it sounds like we no longer need a platform context at all if we're in your new mode, so you should not need the layer check here. Also, the *not* enabled case should be `true`, not false, right? (we need a platform context to paint selections without the new API)

Reply:
Ah yes, the _not_ enabled case should return `true`.

Reply:
Okay, since none of the layers are WP-backed, I'll just remove the layer check.

Reply:
Addressed in https://github.com/WebKit/WebKit/commit/ab434ecd8a237eb5256d62fbc8744be2db5813fc